testing/zerotier-one/zerotier-one.pre-install

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Do not fail if user exists, since lbu backup can be restored in diskless with /var/lib/zerotier-one credentials
+
+id zerotier-one >/dev/null 2>&1 || /usr/sbin/adduser -D -H -S -h /var/lib/zerotier-one -G netdev zerotier-one
```

Comment:
This is unnecessary complicated, please follow the same pattern as used in other packages.

```sh
#!/bin/sh

adduser -D -H -S -h /var/lib/zerotier-one -s /sbin/nologin -G netdev -g zerotier-one zerotier-one 2>/dev/null

exit 0
```